eventhubs-capture-synapse-delta/README.md

```diff
@@ -0,0 +1,179 @@
+---
+topic: sample
+languages:
+  - azurecli
+  - avro
+  - python
+products:
+  - azure
+  - azure-container-instances
+  - azure-databricks
+  - azure-event-hubs
+  - azure-storage
+statusNotificationTargets:
+  - algattik@microsoft.com
+---
+
+# Streaming at Scale with Azure Storage, Synapse and Delta Lake
+
+This sample uses Azure Synapse to ingest EventHub-captured blobs into Azure Storage [Delta Lake](https://docs.azuredatabricks.net/delta/index.html) storage.
+
+Azure Container Instances generates sample data into Event hub which are then saved in Azure Blob Storage using [Event Hub Capture](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-capture-overview). A blob landing in the designated container triggers a Synapse pipeline which processes and stores it in a Delta table.
+The provided scripts will deploy an end-to-end solution complete with load test client.
+
+## Running the Scripts
+
+Please note that the scripts have been tested on [Ubuntu 18 LTS](http://releases.ubuntu.com/18.04/), so make sure to use that environment to run the scripts. You can run it using Docker, WSL or a VM:
+
+- [Ubuntu Docker Image](https://hub.docker.com/_/ubuntu/)
+- [WSL Ubuntu 18.04 LTS](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q?activetab=pivot:overviewtab)
+- [Ubuntu 18.04 LTS Azure VM](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuServer1804LTS)
+
+The following tools/languages are also needed:
+
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest)
+  - Install: `sudo apt install azure-cli`
+- [jq](https://stedolan.github.io/jq/download/)
+  - Install: `sudo apt install jq`
+- [python](https://www.python.org/)
+  - Install: `sudo apt install python python-pip`
+
+## Setup Solution
+
+Make sure you are logged into your Azure account:
+
+    az login
+
+and also make sure you have the subscription you want to use selected
+
+    az account list
+
+if you want to select a specific subscription use the following command
+
+    az account set --subscription <subscription_name>
+
+once you have selected the subscription you want to use just execute the following command
+
+    ./create-solution.sh -d <solution_name> -p <synapse_sql_password> -w <wait_verfication>
```

Comment:
This isn't necessarily a bad thing (it might even be better), but just calling out a difference. The script for eventhubs-capture-databricks-delta creates a resource group named the same as solution_name whereas this creates solution_name-rg.